api-client/src/endpoints/createCitiesEndpoint.ts

```diff
@@ -36,7 +37,10 @@ export default (baseUrl: string): Endpoint<void, Array<CityModel>> =>
               prefix: city.prefix,
               longitude: city.longitude,
               latitude: city.latitude,
-              aliases: city.aliases
+              aliases: city.aliases,
+              boundingBox: Array.isArray(city.bounding_box)
+                ? (city.bounding_box.reduce((acc, val) => acc.concat(val), []) as BBox)
```

Comment:
Maybe someone can take a short look how to properly type this. Couldn't find a proper solution instantly